app/src/main/java/com/example/fantasyleague/model/Match.kt

```diff
@@ -1,3 +1,3 @@
 package com.example.fantasyleague.model
 
-data class Match(var teams: Pair<Team,Team>? = null, var isOver: Boolean = false)
+data class Match(var teams: Pair<Team,Team>? = null, var isOver: Boolean = false, var result: MatchResultInfo?=null, var matchInfo: String?= null, var time: String?=null)
```

Comment:
Convert it into two or three lines. Use the line break in android studio for reference